src/main/java/mate/academy/service/impl/MovieServiceImpl.java

```diff
@@ -26,4 +26,16 @@ public Movie get(Long id) {
     public List<Movie> getAll() {
         return movieDao.getAll();
     }
+
+    @Override
+    public Boolean update(Movie movie) {
```

Comment:
Consider changing the return type of the `update` method to `void`. The current approach of returning a boolean by comparing entities is not reliable for determining if the update was successful.